scss/components/_tagsBox.scss

```diff
@@ -65,6 +65,7 @@
 		flex-grow: 1;
 		margin: 0 2px;
 		//width: $item-pane-width - $icon-width - $delete-button-width - ($li-side-margin * 2);
+		width: 0;
```

Comment:
Add comment to explain this?

Reply:
Added. Basically `width: 0` and `height: 0` are need to make flexbox in XUL work more similar to regular flexbox. Otherwise child container forces its width/height to parent container by default, which never happens in regular HTML.